samples/highcharts/demo/heatmap-bubbles/demo.js

```diff
@@ -0,0 +1,106 @@
+// Create the chart
+Highcharts.chart('container', {
+
+    title: {
+        text: 'Riskmap',
+        style: {
+            fontSize: '1em'
+        }
+    },
+
+    xAxis: {
+        categories: ['Rare', 'Unlikely', 'Possible', 'Likely', 'Almost Certain'],
+        min: 0,
+        max: 4
+    },
+
+    yAxis: {
+        categories: ['Insignificant', 'Minor', 'Moderate', 'Major', 'Critical'],
+        title: null,
+        min: 0,
+        max: 4
+    },
+
+    chart: {
+        marginTop: 40,
+        marginBottom: 80,
+        plotBorderWidth: 1
+    },
+
+    tooltip: {
+        format: '<b>{point.z}</b> Vulnerabilities'
+    },
+    colorAxis: {
+        stops: [
+            [0, '#FF0000'],
+            [0.5, '#FFFF00'],
+            [1, '#00FF00']
+        ]
+    },
+
+    legend: {
+        enabled: false
+    },
+
+    plotOptions: {
+        series: {
+            states: {
+                inactive: {
+                    opacity: 1
+                }
+            }
+        }
+    },
+
+    series: [{
+        // Heatmap
+        type: 'heatmap',
+        name: 'Criticality of data',
+        borderWidth: 1,
+        color: '#FFFFFF',
+        data: [[0, 4, 3], [1, 4, 2], [2, 4, 1], [3, 4, 0], [4, 4, 0],
+            [0, 3, 4], [1, 3, 3], [2, 3, 2], [3, 3, 1], [4, 3, 0],
+            [0, 2, 6], [1, 2, 5], [2, 2, 4], [3, 2, 2], [4, 2, 1],
+            [0, 1, 8], [1, 1, 7], [2, 1, 6], [3, 1, 5], [4, 1, 3],
+            [0, 0, 10], [1, 0, 10], [2, 0, 8], [3, 0, 6], [4, 0, 5]],
+        dataLabels: {
+            enabled: false
+        },
+        tooltip: {
+            enabled: false
+        },
```

Comment:
Disabling the tooltip is not necessary because we have the `enableMouseTracking` option set to false.